program/src/lib.rs

```diff
@@ -14,3 +14,5 @@ pub mod processor;
 pub mod state;
 
 solana_program::declare_id!("Dbf7u6x15DhjMrBMunY3XoRWdByrCCt2dbyoPrCXN6SQ");
+
+pub const LOCKUP_COOLDOWN_SECONDS: u64 = 30 * 60; // 30 minutes
```

Comment:
Just curious if we want to have this hard-coded like this? I'm assuming the 30 minute value came from some doc or discussion with the Paladin team, but how would anyone go about changing this timer? I guess a governance-gated program upgrade is the way.

Alternatively, this program can use a config (like the stake program).